gqlerrors/error.go

```diff
@@ -38,6 +42,9 @@ func NewError(message string, nodes []ast.Node, stack string, source *source.Sou
 	}
 	if len(positions) == 0 && len(nodes) > 0 {
 		for _, node := range nodes {
+			if node == nil || reflect.ValueOf(node).IsNil() {
```

Comment:
For me, it's cleaner to separate checking if node is nil from checking if an attribute (or something that GetLoc() returns) is nil. Also, three conditions within the single line is quite too much.